### ISSUE TITLE
Crashes.

### ISSUE THREAD

**Issue by MagmaSlime123** [Fuzss/fivefeetsmall#59]
See the report. Modlist/Versions are in it.

[crash-2021-12-31_14.31.22-client.txt](https://github.com/Fuzss/aquaacrobatics/files/7797066/crash-2021-12-31_14.31.22-client.txt)

**Comment by mxnmnm** [comment]
Install Mixin 0.7-0.8 Compatibility

**Comment by embeddedt** [comment]
Can you provide the whole log (`latest.log`)? I think there was another error that the crash report didn't capture.

**Comment by MagmaSlime123** [comment]
[2021-12-31-4.log.gz](https://github.com/Fuzss/aquaacrobatics/files/7797198/2021-12-31-4.log.gz)

Hopefully this is the right one.

**Comment by embeddedt** [comment]
It looks like the crash is caused by Phosphor, not Aqua Acrobatics. However, as an educated guess, the crash may be related to MixinBootstrap providing a newer version of Mixin than what Phosphor is expecting. You can try using an [older version](https://www.curseforge.com/minecraft/mc-mods/mixinbootstrap/files/3076610) or removing it and installing Mixin 0.7-0.8 Compatibility instead.

**Comment by MagmaSlime123** [comment]
MixinBootstrap and Phosopher run fine together. AquaAcrobatics is what causes the crash.

**Comment by embeddedt** [comment]
Are you sure you got the right log? The error there said that a chunk mixin in Phosphor failed to apply, and there is no mixin in Aqua Acrobatics I see that could conflict with that (it doesn't touch chunks at all).

For the record, I use Phosphor 0.2.7 and Aqua Acrobatics in a private modpack and they work just fine together, so I think there must be a third mod somewhere that introduces this issue. I don't actually need MixinBootstrap installed in that case as I have other mods which have a Mixin implementation. I'll test it when I get a chance, however.

**Comment by embeddedt** [comment]
I tested locally, and as I suspected, the crash occurs with only MixinBootstrap 1.1.0 and Phosphor. However, using the older version of MixinBootstrap does fix the crash.

Phosphor would need to be fixed to work with the newer Mixin version, and while I could do this, it seems risky as there are likely other mods which would break in a similar manner. I'd suggest simply using an older version of MixinBootstrap for now; I don't think anything in 1.12 depends on a Mixin version newer than 0.8.2.

EDIT: Looks like this is a known issue. https://github.com/LXGaming/MixinBootstrap/issues/52
